crates/matrix-sdk/src/event_cache/deduplicator.rs

```diff
@@ -267,4 +267,33 @@ mod tests {
             assert!(events.next().is_none());
         }
     }
+
+    #[test]
+    fn test_bloom_filter_growth() {
+        // This test was used as a testbed to observe, using `valgrind --tool=massive`,
+        // the total memory allocated by the deduplicator. We keep it checked in
+        // to revive this experiment in the future, if needs be.
+
+        let num_rooms = 10;
+        let num_events = 100;
+        //let num_rooms = std::env::var("ROOMS").unwrap().parse().unwrap();
+        //let num_events = std::env::var("EVENTS").unwrap().parse().unwrap();
+
+        let mut dedups = Vec::with_capacity(num_rooms);
```

Comment:
This isn't really needed for the benchmark, right? We don't do anything with the result that gets pushed.

Reply:
Oh wait, it's probably just to know how much memory is used, never mind.

Reply:
Exactly!